src/swap/types.ts

```diff
@@ -78,6 +85,13 @@ export type GetSwapMessageParams = {
   toToken?: Token;
 };
 
+/**
+ * Note: exported as public Type
```

Comment:
@cpcramer it says `exported as public Type`, but in this PR is not either exported or documented.

That comment has a deep meaning, which is not reflected in the code you wrote.

If you add that comment, than you want to make sure to export and document the type in `sitet`.

Reply:
Got it. Will fix!